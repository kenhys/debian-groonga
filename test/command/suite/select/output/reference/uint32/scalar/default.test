table_create Ages TABLE_PAT_KEY UInt32

table_create Users TABLE_PAT_KEY ShortText
column_create Users age COLUMN_SCALAR Ages

load --table Users
[
["_key"],
["alice"],
["bob"]
]

select Users
