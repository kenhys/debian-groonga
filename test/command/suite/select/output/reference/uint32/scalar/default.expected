table_create Ages TABLE_PAT_KEY UInt32
[[0,0.0,0.0],true]
table_create Users TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
column_create Users age COLUMN_SCALAR Ages
[[0,0.0,0.0],true]
load --table Users
[
["_key"],
["alice"],
["bob"]
]
[[0,0.0,0.0],2]
select Users
[
  [
    0,
    0.0,
    0.0
  ],
  [
    [
      [
        2
      ],
      [
        [
          "_id",
          "UInt32"
        ],
        [
          "_key",
          "ShortText"
        ],
        [
          "age",
          "Ages"
        ]
      ],
      [
        1,
        "alice",
        0
      ],
      [
        2,
        "bob",
        0
      ]
    ]
  ]
]
